project/Dependencies.scala

```diff
@@ -9,18 +9,19 @@ import sbt._
 
 object Version {
   // OSS
-  val akka      = "2.4.10"
-  val akkaDns   = "2.4.2-M1"
-  val lagom     = "1.1.0"
-  val scalaTest = "3.0.0"
+  val akka       = "2.4.20"
+  val akkaDns    = "2.4.2"
+  val lagom13    = "1.3.7"
+  val scalaTest  = "3.0.1"
 }
 
 object Library {
   // OSS
-  val akkaDns     = "ru.smslv.akka"       %% "akka-dns"          % Version.akkaDns
-  val akkaTestkit = "com.typesafe.akka"   %% "akka-testkit"      % Version.akka
-  val lagom       = "com.lightbend.lagom" %% "lagom-javadsl-api" % Version.lagom
-  val scalaTest   = "org.scalatest"       %% "scalatest"         % Version.scalaTest
+  val akkaDns            = "ru.smslv.akka"       %% "akka-dns"              % Version.akkaDns
+  val akkaTestkit        = "com.typesafe.akka"   %% "akka-testkit"          % Version.akka
+  val lagom13JavaClient  = "com.lightbend.lagom" %% "lagom-javadsl-client"  % Version.lagom13
+  val lagom13ScalaClient = "com.lightbend.lagom" %% "lagom-scaladsl-client" % Version.lagom13
```

Comment:
Yeah, but you wouldn't want to mix Lagom client 1.3.7 with, say Lagom server 1.3.5.

I thought we had changed the Lagom dependencies in `conductr-lib` to `provided` for this reason, but I checked again and it looks like not. We definitely discussed it at some point.

I guess it was tried and rejected in https://github.com/typesafehub/conductr-lib/pull/129